### ISSUE TITLE
dnd5e 3.1. Items inside containers maintain there equipped status ignoring the container equipped status

### ISSUE THREAD

**Issue by tposney** [foundryvtt/dnd5e#2908]
It's not clear to me if this is intentional.
The impact is that passive effects from items in containers apply according to the item's equipped status, rather than the containers equipped status.

Should the item equipped status be overridden by the containers equipped status?

**Comment by Fyorl** [comment]
I think having contained items inherit the equipped status from their parent container was not something that was ever considered. I suppose it would make sense if we do #2801 and treat unequipping a container as 'dropping' it.

**Comment by schultzcole** [comment]
I don't think it makes sense for items in containers to inherit the container's equipped status. If I have a magic helmet in a backpack and I'm wearing the backpack, I'm not also wearing the helmet and I shouldn't get the effects from the helmet.

**Comment by unsoluble** [comment]
Surely the equipped status of all items inside a container should not be toggled by ... yes, what cole said :)

**Comment by Fyorl** [comment]
Yes you are right, unequipping a container should probably force contained items to not be considered equipped, but the other way around needs to allow for individual items to be manually toggled.

**Comment by unsoluble** [comment]
Is there any case in which an item inside a container could be considered `equipped` for the purposes of rule mechanics? I can't think of one.

**Comment by unsoluble** [comment]
Ahhh interesting! So we do need to allow toggling `equipped` back on as needed. But yeah I do still think it's reasonable to default to `unequipped` on container additions. And considering things like the Warning weapon, toggling the equipped status of the container itself probably shouldn't propagate down to the contained items at all, leaving them as-is.

**Comment by krbz999** [comment]
There should probably be enhanced logic to determine whether an item can apply its AE despite being in a container (or should care), not just determined from its `equipped` and `attunement` state.

**Comment by Fyorl** [comment]
> (Or I guess maybe another one-way, unequipping everything inside if the container is unequipped, but not equipping everything inside on equip.)

I think when we prepare contained item data we would override their derived equipped state to be unequipped if their container is also unequipped, but not vice versa. That's probably the easiest way to do it.

> There should probably be enhanced logic to determine whether an item can apply its AE despite being in a container (or should care), not just determined from its `equipped` and `attunement` state.

That would be another way to solve it but it would require extra data in the data model and additional UI to configure it, which seems overkill for such a niche mechanic. Using the equipped state for this purpose seems good enough to me at least.
